### PR TITLE
Add support for pkgr builds

### PR DESCRIPTION
**Pull request by rbclark**
Cypress uses pkgr for building a .deb package for our distributed VMs. In order to instruct pkgr to build as we need, we need to add a Procfile, .pkgr.yml, and specify the supported engine versions (if we don't specify them it defaults to node 6 which fails).

### CODE REVIEW

File: .pkgr.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+targets:
```
Comment (mayerm94) [inline .pkgr.yml]: Would it be possible to put this file in a top-level config/ directory within the project?

Comment (rbclark) [reply .pkgr.yml]: Packager requires it to be here or it won't pick it up.

Comment (lmd59) [reply .pkgr.yml]: I think I'm okay with this being at the top level because it's a ".-" hidden file.

Comment (rbclark) [reply .pkgr.yml]: Just double checked, the .pkgr.yml is the only supported way I can find: http://doc.packager.io/documentation/customizing-the-build/

**Pull request merged**